### PR TITLE
Expand Archive Instructions

### PR DESCRIPTION
**Pull request by happywillow0**
Tested how archives are read by Tachi. Expanded the section to explain how to use an archive file.

### CODE REVIEW

File: src/help/guides/reading-local-manga.md [hunk 1]
```diff
@@ -76,6 +76,38 @@ You can also place your chapters or manga in both storages and Tachiyomi will me
 
 Tachiyomi will see four chapters in a single manga.
 
-The path to the folder (or archive) with images must contain both the manga title and the chapter name (as seen above).
+The path to the folder with images must contain both the manga title and the chapter name (as seen above). 
+
+Folders inside archives such as `ZIP`/`CBZ` are not currently supported. You must place the archive inside the manga folder where the name will become the chapter title and all the images inside will become pages for that chapter. 
+
+::: tip ZIP EXAMPLE
+<div class="side-by-side">
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: Don't need this.

File: src/help/guides/reading-local-manga.md [hunk 2]
```diff
@@ -76,6 +76,38 @@ You can also place your chapters or manga in both storages and Tachiyomi will me
 
 Tachiyomi will see four chapters in a single manga.
 
-The path to the folder (or archive) with images must contain both the manga title and the chapter name (as seen above).
+The path to the folder with images must contain both the manga title and the chapter name (as seen above). 
+
+Folders inside archives such as `ZIP`/`CBZ` are not currently supported. You must place the archive inside the manga folder where the name will become the chapter title and all the images inside will become pages for that chapter.
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: Maybe add a heading before this, like "Archive files"?

File: src/help/guides/reading-local-manga.md [hunk 3]
```diff
@@ -76,6 +76,38 @@ You can also place your chapters or manga in both storages and Tachiyomi will me
...
+					</ul>
+				</li>
+				<li>&hellip;</li>
+			</ul>
+		</li>
+	</ul>
+</div>
+:::
 
 You can then access the manga in `Catalogues > Local manga`.
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: If you do add a heading, it'd flow better if this was still with the previous section of content.

File: src/help/guides/reading-local-manga.md (hunk 1 above)
Comment (happywillow0) [reply src/help/guides/reading-local-manga.md]: Removed div side by side
Yea, that was direct copy and paste. 

Unless you mean the entire example.

Comment (arkon) [reply src/help/guides/reading-local-manga.md]: Just meant that div. I think the indentation leftover is causing it to be rendered as code though.

File: src/help/guides/reading-local-manga.md (hunk 2 above)
Comment (happywillow0) [reply src/help/guides/reading-local-manga.md]: Thanks for the advise! Done

File: src/help/guides/reading-local-manga.md (hunk 1 above)
Comment (happywillow0) [reply src/help/guides/reading-local-manga.md]: I put back the div block but left out the side by side class.

Comment (happywillow0) [reply src/help/guides/reading-local-manga.md]: No wait, I guess I don't need it. Fixed with the first set of indents.

File: src/help/guides/reading-local-manga.md [hunk 4]
```diff
@@ -19,6 +19,12 @@ Supported chapter formats are directories with pictures inside, `ZIP`/`CBZ`, `RA
 
 You can also place your chapters or manga in both storages and Tachiyomi will merge them.
 
+You can then access the manga in `Catalogues > Local manga`.
+
+## Folder Structure
+
+Tachiyomi requires a specific folder structure for local mangas to be correctly processed. Local mangas will be read from the `Tachiyomi/local`folder. Each Manga musst have a `Manga` folder and a `Chapter`folder. Images will then go into the chapter folder. See below for more information on archive files. You can refer to the following example:
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: - `local mangas` -> `local manga`
- `/localfolder` -> `/local folder`
- `Each Manga musst` -> `Each manga must`
- `Chapterfolder` -> `Chapter folder`

File: src/help/guides/reading-local-manga.md [hunk 5]
```diff
@@ -76,6 +82,42 @@ You can also place your chapters or manga in both storages and Tachiyomi will me
 
 Tachiyomi will see four chapters in a single manga.
 
-The path to the folder (or archive) with images must contain both the manga title and the chapter name (as seen above).
+The path to the folder with images must contain both the manga title and the chapter name (as seen above). 
+
+## Archive Files
+
+Archive files such as `ZIP`/`CBZ` are supported but the folder structure inside is not. Any folers inside the archive file are ignored. You must place the archive inside the `Manga` folder where the name will become the `Chapter` title. All images inside the archive regardless of folder structure will become pages for that chapter.
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: - `folers` -> `folders`

File: src/help/guides/reading-local-manga.md (hunk 4 above)
Comment (happywillow0) [reply src/help/guides/reading-local-manga.md]: Thanks 😓

**Pull request merged**